docs/user-guide/advanced-configuration/deploying-a-specific-version.md

```diff
@@ -5,9 +5,9 @@ There are a few variables that are customizable for eda the image management.
 | Name                   | Description               | Default                                 |
 | ---------------------- | ------------------------- | --------------------------------------  |
 | image                  | Path of the image to pull | quay.io/ansible/eda-server              |
-| image_version          | Image version to pull     | latest                                  |
+| image_version          | Image version to pull     | value of DEFAULT_EDA_VERSION or main    |
 | image_web              | Path of the image to pull | quay.io/ansible/eda-ui                  |
-| image_web_version      | Image version to pull     | latest                                  |
+| image_web_version      | Image version to pull     | value of DEFAULT_EDA_UI_VERSION or main |
```

Comment:
the tags in the builds as part of release use the `latest` tag

Reply:
@jon-nfc the `image_version` and `image_web_version` parameters are referring to the tags for the application images ([eda-server](quay.io/ansible/eda-server) and [eda-ui](quay.io/ansible/eda-ui)), not the operator.  

The app images params default to `main` in the development branch (`main`), but in released operators, they will be pinned to the version of the application image tested at the time the operator was released.

So we need to stick with `main` for the app images. Also, the eda-server repo does not have a `latest` tag.